R/betydb.R

```diff
@@ -186,14 +139,11 @@ betydb_auth <- function(x,y,z){
   }
 }
 
-makeurl <- function(x, fmt){
-  fmt <- match.arg(fmt, c("json","xml","csv"))
-  paste0(betyurl(), paste0(x, "."), fmt)
-}
-
-makeidurl <- function(x, id, fmt){
+makeurl <- function(x, id, fmt){
```

Comment:
In <code>makeurl</code> if <code>id</code> is null it will return a url like <code>betydb.org/table.json</code>, otherwise it will make a url like <code>betydb.org/table/id.json</code>. The url formats are different so <code>id</code> is needed in this function.